photonVision/cameras/Microsoft_LifeCam_HD-3000/pipelines/Cargo Blue.json

```diff
@@ -0,0 +1,86 @@
+[ "ColoredShapePipelineSettings", {
+  "pipelineIndex" : 0,
+  "pipelineType" : "ColoredShape",
+  "inputImageFlipMode" : "NONE",
+  "inputImageRotationMode" : "DEG_0",
+  "pipelineNickname" : "Cargo Blue",
+  "cameraExposure" : 39.4,
+  "cameraBrightness" : 50,
+  "cameraGain" : -1,
+  "cameraVideoModeIndex" : 0,
+  "streamingFrameDivisor" : "NONE",
+  "ledMode" : true,
+  "inputShouldShow" : true,
+  "outputShouldShow" : true,
+  "hsvHue" : {
+    "first" : 91,
+    "second" : 140
+  },
+  "hsvSaturation" : {
+    "first" : 58,
+    "second" : 255
+  },
+  "hsvValue" : {
+    "first" : 27,
+    "second" : 255
+  },
```

Comment:
Since the hue range for red wraps from like 190->20 we need to set the lower to be 20 and the upper to 190 and mark it as inverted. **This will be possible once we get PhotonVision updated with the following change**: https://github.com/PhotonVision/photonvision/pull/413/

![image](https://user-images.githubusercontent.com/3326002/152269319-81999470-9dcc-4518-bdb5-5fd7f3220c31.png)
![image](https://user-images.githubusercontent.com/3326002/152269495-6bdc484d-f6e6-451a-9d21-a50acefa25f9.png)

You can see red hue range actually begins at the end, and wraps around to the beginning.

```suggestion
  "hsvValue" : {
    "first" : 27,
    "second" : 255
  },
  "hsvHueShouldInvert" : true,
```